### ISSUE TITLE
Non-ascii paths are not supported

### ISSUE THREAD

**Issue by Yanpas** [mbusb/multibootusb#33]
For example for path `/home/user/Дистрибутивы/` it will be impossible to open iso. 
Terminal output:

```
File not selected...
Traceback (most recent call last):
  File "/usr/bin/multibootusb", line 219, in browse_iso
    config.iso_link = str(QtGui.QFileDialog.getOpenFileName(self, 'Select an iso...', os.path.expanduser('~'), 'ISO Files (*.iso)'))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 13-21: ordinal not in range(128)
can't open Exception AttributeError: "ISO9660 instance has no attribute 'isoFile'" in <bound method ISO9660.__del__ of <scripts.isodump.ISO9660 instance at 0x7f6270452e60>> ignored
```

**Comment by ricflomag** [comment]
Confirmed.

**Comment by lu6fer** [comment]
You can update multibootusb.py lines 230 to 235 like this :
```python
config.iso_link = unicode(QtGui.QFileDialog.getOpenFileName(self, 'Select an iso...', os.path.expanduser('~'), 'ISO Files (*.iso)'))
            if config.iso_link:
                if platform.system() == "Windows":
                    if "/" in config.iso_link:
                        config.iso_link = config.iso_link.strip().replace("/", "\\")
                self.ui.lineEdit.insert(unicode(config.iso_link)) 
```

**Comment by mbusb** [comment]
Fixed in the latest version 8.0.0.
